mkeditorpacks/README.md

```diff
@@ -0,0 +1,19 @@
+MkEditorPacks
+=============
+
+This a tool suite to build game packs for map editors.
+
+It is developped for the [Unvanquished](https://unvanquished.net/) project, but must fit any project supported by the editors listed below.
+
+Supported editors
+-----------------
+
+- [GtkRadiant](http://icculus.org/gtkradiant/)
+- [NetRadiant](https://gitlab.com/xonotic/netradiant)
+
+Authors
```

Comment:
@Viech you can change the license to any suitable permissive or copyleft license.